plugins/Sf2Player/Sf2Player.cpp

```diff
@@ -683,10 +714,10 @@ void Sf2Instrument::playNote( NotePlayHandle * _n, sampleFrame * )
 		pluginData->midiNote = midiNote;
 		pluginData->lastPanning = 0;
 		pluginData->lastVelocity = _n->midiVelocity( baseVelocity );
-		pluginData->fluidVoice = nullptr;
 		pluginData->isNew = true;
 		pluginData->offset = _n->offset();
 		pluginData->noteOffSent = false;
+		pluginData->panning = std::clamp(PanningLeft, _n->getPanning(), PanningRight);
```

Comment:
I believe ``_n->getPanning()`` should be the first argument?
```suggestion
		pluginData->panning = std::clamp(_n->getPanning(), PanningLeft, PanningRight);
```

Reply:
It should indeed. I think I can remove the clamp instead though - it looks like `Note` enforces the bound itself.